.circleci/config.yml

```diff
@@ -64,7 +65,15 @@ jobs:
             kubectl get all -o wide --all-namespaces
             kubectl cluster-info dump > /go/out/logs/cluster-info.dump.txt
             kubectl describe pods -n istio-system > /go/out/logs/pods-system.txt
-            /go/bin/go-junit-report </go/out/tests/test-report-noauth-simple.raw > /go/out/tests/test-report-noauth-simple.xml
+            /go/bin/go-junit-report </go/out/tests/test-report-noauth-simple.raw > /go/out/tests/junit_noauth-simple.xml
+      - run:
+          name: bin/ci_to_gubernator.sh --job_finishes
+          when: always
+          command: |
```

Comment:
Please consider updating the  defaults: &defaults to do that for every job.

Reply:
I looked into the pre, post special keys for circle ci but turns out they are supported only in circleci version 1, while we are using version 2. I further simplified the way of getting exit_code  and also extracted `initWorkingDir` to its own yaml anchor just like &default.